index.js

```diff
@@ -0,0 +1,16 @@
+let names=[]
+function myFunction()
+{
+ let x=document.getElementById("box");
+  names.push(document.getElementById("name").value);
+ x.innerHTML=names.join('<br/><br/>'); 
+}
+
+//for accessibility
+let input = document.getElementById("name");
```

Comment:
Since this variable isn't changing, we should use `const` instead of `let`. Please refer to this [article](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/) to understand the difference

Reply:
Alright, sir.